src/storage/src/hummock/compactor.rs

```diff
@@ -609,6 +610,23 @@ impl Compactor {
         }
     }
 
+    async fn start_uploading_task_coordinator(
+        sstable_store: Arc<SstableStore>,
+        mut receiver: mpsc::UnboundedReceiver<UploadRequest>,
+    ) {
+        while let Some(request) = receiver.recv().await {
+            let ret = do_upload(
```

Comment:
Seems that we are serializing all the upload here. A previous benchmark seems to show that if we are not able to parallelize the upload, we cannot make good use of the IO bandwidth and this will increase the latency for committing an epoch. 

I think we should buffer the upload requests in a local variable and limit the total concurrent uploading size.

Reply:
> Seems that we are serializing all the upload here. A previous benchmark seems to show that if we are not able to parallelize the upload, we cannot make good use of the IO bandwidth and this will increase the latency for committing an epoch.
> 
> I think we should buffer the upload requests in a local variable and limit the total concurrent uploading size.

each requests will firstly watch current upload size, if current upload size doesn't exceed limit, it will not goto here